virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -217,6 +215,13 @@ link:https://access.redhat.com/support/offerings/techpreview[Technology Preview
 * The Pod Disruption Budget (PDB) prevents pod disruptions for migratable virtual machine images. If the PDB detects pod disruption, then `openshift-monitoring` sends a `PodDisruptionBudgetAtLimit` alert every 60 minutes for virtual machine images that use the `LiveMigrate` eviction strategy. (link:https://issues.redhat.com/browse/CNV-33834[*CNV-33834*])
 ** As a workaround, xref:../../monitoring/managing-alerts.adoc#silencing-alerts_managing-alerts[silence alerts].
 
+[discrete]
+[id="virt-4-15-ki-networking"]
+==== Networking
+//CNV-38593: RN: Known issue Remove invalid annotation from "OVN Kubernetes secondary localnet network"
+* The invalid annotation `k8s.v1.cni.cncf.io/resourceName: openshift.io/` appears during the creation of a Network Access Device (NAD) with the *OVN Kubernetes secondary localnet network* network type. This annotation prevents the starting of the VM.
```

Comment:
```suggestion
* When creating a network attachment definition (NAD) for an OVN-Kubernetes localnet topology by using the web console, the invalid annotation `k8s.v1.cni.cncf.io/resourceName: openshift.io/` appears. This annotation prevents the starting of the VM.
```

Reply:
I looked at this in the console, and the **OVN Kubernetes secondary localnet network** appears at the  network type. Does it need to change?

Reply:
![OVN Kubernetes network types](https://github.com/openshift/openshift-docs/assets/45106934/2ef7427e-4755-4bd9-aaa0-2b6c58f64b14)

Reply:
Here is the web console procedure for creating a localnet NAD: https://docs.openshift.com/container-platform/4.15/virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.html#virt-creating-nad-localnet-console_virt-connecting-vm-to-ovn-secondary-network. 
If you want to refer to the OVN-Kubernetes secondary network in a general way, remove the bold formatting. Otherwise, IMO, you will need to provide more context about where this field is in the UI.